server/front/src/index.ts

```diff
@@ -276,28 +278,48 @@ export function start (
 
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
   app.post('/files', async (req, res) => {
+    function isValidFile (file: UploadedFile): boolean {
+      const maxSize = Number(config.uploadMaxFileSize)
+      if (isNaN(maxSize)) {
+        throw new Error('Invalid uploadMaxFileSize value')
+      }
+      return file.size <= maxSize && config.uploadAllowedMimeTypes.includes(file.mimetype)
```

Comment:
For better user experience it would be nice to pass allowed mime types to file inputs so users could not choose files of not allowed types.